File: account/src/main/java/serp/project/account/core/domain/dto/request/GetUserParams.java
/**
 * Author: QuanTuanHuy
 * Description: Part of Serp Project
 */

package serp.project.account.core.domain.dto.request;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@SuperBuilder
public class GetUserParams extends BaseGetParams{
    private String search;
}

File: api_gateway/src/core/domain/dto/request/account/get_user_params.go
/*
Author: QuanTuanHuy
Description: Part of Serp Project
*/

package request

type GetUserParams struct {
	Page           *int    `json:"page,omitempty"`
	PageSize       *int    `json:"pageSize,omitempty"`
	SortBy         *string `json:"sortBy,omitempty"`
	SortDir        *string `json:"sortDir,omitempty"`
	Search         *string `json:"search,omitempty"`
	OrganizationID *int64  `json:"organizationId,omitempty"`
	Status         *string `json:"status,omitempty"`
}
